requirements.txt

```diff
@@ -6,13 +6,13 @@
 # __author__ = fla
 
 django==1.5.5
-nose==1.3.0
+nose==1.3.6
```

Comment:
Is nose a requirement for production? Guess not, so please include this in `requirements_dev.txt`